external-api/src/v1/errors/invalid-entity-type.error.ts

```diff
@@ -0,0 +1,20 @@
+import { ERRORS } from '../../constants';
+
+export class InvalidEntityTypeError extends Error {
+  /**
+   * Represents an error that occurs when an invalid entity type is encountered.
+   * @param entityType - Entity type in string
+   */
+  constructor(entityType: string) {
+    super();
+    this.name = this.constructor.name;
+    this.message = ERRORS.ENTITY_TYPE.INVALID;
+    this.cause = `Invalid entity type: ${entityType}`;
+
+    /**
+     * Needed when compiling against ES5 to allow us to call `instanceof` on custom errors
```

Comment:
What a horrible bug in es5 lol

Reply:
Will update to `ES6` in a separate PR for consistency as currently its dispersed.